.github/workflows/CompatHelper.yml

```
@@ -6,6 +6,8 @@ on:
 jobs:
   CompatHelper:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

```suggestion
    permissions:
      contents: write
```
It could be that we can drop COMPATHELPER_PRIV if we set
```yaml
permissions:
    contents: write
    pull-requests: write
```
I've tested this and it didn't work. This could be due to:

1. GITHUB_TOKEN not having enough permissions open a PR
2. CompatHelper not reading the GITHUB_TOKEN even when permissions are set

---

You need the `COMPATHELPER_PRIV` to get GitHub Actions CI to automatically trigger on PRs opened by CompatHelper.

---

Yes. Sounds like it